### DISCUSSION TITLE
Add option to show a comment's parent

### DISCUSSION THREAD

**Discussion by marinmo** [Docile-Alligator/Infinity-For-Reddit#877] [Ideas]
Hello,

While browsing long threads, it's sometimes hard to remember the exact wording/message of the comment you're reading the reply to. Therefore it'd be really convenient to have an option to view the current comment's parent. An example screenshot of the functionality taken from Sync for reddit;
![Show parent comment](https://user-images.githubusercontent.com/15056491/174732991-cfb19e19-d26b-421c-8ec4-5aa6e45b997a.jpg)

Thanks in advance for your consideration!

**Comment by Wladefant** [comment]
Definitely worth it

**Reply by different55** [reply]
Would be very nice to be able to navigate back up a comment chain, too. Right now if you want to see the comment that's only *just* fallen outside of context range it seems like you have to backtrack all the way to the top level comments and find your way from there.
